README.md

```diff
@@ -187,8 +187,8 @@ specified in the model's configuration using the 'backend' and
 'platform' settings. Depending on the backend one or the other of
 these properties is optional.
 
-* For TensorRT, 'platform' must be set to *tensorrt_plan*. Currently,
```

Comment:
@Tabrizian probably a good idea to add the python backend in the [backends section](https://github.com/triton-inference-server/backend/blob/main/README.md#backends) as well.

@tanmayv25 same for the onnxruntime backend as well.

Reply:
The backends listed here are for older backends that can still be selected using the "platform" setting. Newer backends (like python and onnxruntime) do not use "plaform" and so do not need to be listed here.

Reply:
Ah yes! Resolving this